src/keria/app/agenting.py

```diff
@@ -297,7 +171,7 @@ class Agency(doing.DoDoer):
 
     def __init__(self, name, bran, base="", releaseTimeout=None,
                  configFile=None, configDir=None, adb=None, temp=False,
-                 curls=None, iurls=None, durls=None):
```

Comment:
This was exposed by the end to end testing I did. Not passing through the `cf` arg prevented the configuration file I used in testing  from properly propagating through to each of the Agents. Passing the cf in here aligns the Agency with similar patterns in Habery and Hab.